### DISCUSSION TITLE
Unable to Bundle Install with ruby 3.1.2 and Rails 7.0.2

### DISCUSSION THREAD

**Discussion by RailsCod3rFuture** [basecamp/kamal#251] [General]
Greetings, I've been trying to get up and running with MRSK. But there seems to be something wrong with the installation process.

I'm receiving the error below.

```
/Users/dw/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mrsk-0.11.0/lib/mrsk/utils/sensitive.rb:5:in `<class:Sensitive>': uninitialized constant Mrsk::Utils::Sensitive::SSHKit (NameError)

  include SSHKit::Redaction
          ^^^^^^
	from /Users/dw/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/mrsk-0.11.0/lib/mrsk/utils/sensitive.rb:3:in `<main>'
```

**Comment by RailsCod3rFuture** [comment]
gem 'mrsk', '~> 0.10.1' installs perfectly. 0.11.0 has a problem.

**Comment by dhh** [comment]
CC @jeremy, need to add a sshkit require?

**Comment by RailsCod3rFuture** [comment]
Its working on the latest release -- I forgot to mention. Thanks!

**Comment by themoffatt** [comment]
@jeremy - Sorry to reopen an old discussion, but I'm getting this same error again on a fresh install. Using ruby 3.1.2 and version 0.14.0 of the gem. Anything simple I might be missing? The error shows on line 5 of sensitive.rb, which I can see is where that include got moved to.

Edit: I see it also shows line 5 here. heh. I was looking at the next line down in the trace reference line 3.
